### PR TITLE
[QA] Added compatibility with django 2.0

### PR DESCRIPTION
**Pull request by AZtheAsian**
So far this is only halfway - I'm running into a migration error that I'm not clear as to the cause of. (The other errors I do know how to fix, mostly).

### CODE REVIEW

**Comment by bennylope** [comment]
Happy to see this, I started work on this but hadn't had a chance to do much. Will be adding some comments/questions on the review here. Thanks!

**Comment by AZtheAsian** [comment]
@bennylope any idea what the migration error might be? I moved away from testing autoslug bc it's not compatible but idk how that affects it

**Comment by AZtheAsian** [comment]
@bennylope I've fixed it. Review and merge, please?

**Comment by justinmayer** [comment]
@AZtheAsian: This PR should probably include a change to the [relevant documentation](http://django-organizations.readthedocs.io/en/latest/getting_started.html#auto-slug-field) since you have changed the auto-slug dependency. To wit, the following:

> Historically Django-Organizations relied on django-extensions for the base AutoSlugField.

... should perhaps be changed to:

> Django-Organizations relies on django-extensions for the base AutoSlugField.

**Comment by AZtheAsian** [comment]
@justinmayer done-ish?

File: docs/getting_started.rst [hunk 1]
```diff
@@ -85,14 +85,14 @@ URLs defined by that backend. You can do the same with the
...
-Historically Django-Organizations relied on `django-extensions
+Django-Organizations relies on `django-extensions
 <http://django-extensions.readthedocs.org/en/latest/>`_ for the base
 `AutoSlugField
 <http://django-extensions.readthedocs.org/en/latest/field_extensions.html>`_.
 While django-extensions is great, this does require that every project install
 django-extensions for this one small feature.
 
-If you decide to use the default django-organization models by adding
+If you choose not to use the default django-organization models by adding
```
Comment (justinmayer) [inline docs/getting_started.rst]: I'm not sure that wording change is accurate -- I think the original wording may still be true. But my brain is tired. Can you confirm one way or the other, @bennylope?

Comment (nemesisdesign) [reply docs/getting_started.rst]: yes I think so too @AZtheAsian I think is better you cancel that change

**Comment by justinmayer** [comment]
> @bennylope @justinmayer, he's a junior student (under 18 years old) who is learning to contribute to open source projects during and he's doing this task for the Google Code-In with OpenWISPI put up this task because django-organizations is one of the dependencies of openwisp2.

Awesome! Welcome, @AZtheAsian, and many thanks for the contribution. 😁

**Comment by AZtheAsian** [comment]
@nemesisdesign Squashed the commit and undid the change.
@justinmayer I'm really excited about this! I'm glad to contribute.

**Comment by AZtheAsian** [comment]
The Travis fail looks like a pypy issues?

**Comment by pauloxnet** [comment]
Very good work @AZtheAsian I can't see Travis fails

**Comment by bennylope** [comment]
Thanks for putting this together. I took a quick look through and I should have a chance to merge in and cut a new release this weekend.

A point of advice (friendly advice, I want to stress that), don't update the version number when contributing a patch, let the maintainer do that. It might be different than you expect! Thanks again @AZtheAsian, @justinmayer, @nemesisdesign

**Comment by nemesisdesign** [comment]
@bennylope yea good remark! Thanks for your patience, I will approve @AZtheAsian's task on the GCI system so he can move on to other contributions, let us know if you need any more help.

Cheers

**Comment by nemesisdesign** [comment]
👍  thanks

**Comment by bennylope** [comment]
[Version 1.0.0](https://pypi.org/project/django-organizations/) is now on PyPI. Thanks again @AZtheAsian and everyone else.

**Pull request closed without merging**